### PR TITLE
Vocab fixes

### PR DESCRIPTION
**Pull request by kagarmoe**
Signed-off-by: kagarmoe <kgarmoe@chef.io>

## Description

Uses appropriate language. See issue https://github.com/chef/chef-web-docs/issues/3075

## Issues Resolved

[List any existing issues this PR resolves, or any Discourse or
StackOverflow discussion that's relevant]

## Check List

- [ ] Spell Check
- [ ] Local build
- [ ] Examine the local build
- [ ] All tests pass

### CODE REVIEW

File: content/attribute_persistence.md [hunk 1]
```diff
@@ -23,15 +23,15 @@ In Chef Infra Client 16.3 the node Blacklist and Whitelist features were renamed
 
 Legacy config mapping:
 
-- automatic_attribute_blacklist -> blocked_automatic_attributes
```
Comment (tas50) [inline content/attribute_persistence.md]: The point here was to call out the old / new attributes so the old ones have to be in the docs.

File: content/config_rb_client.md [hunk 2]
```diff
@@ -44,7 +44,7 @@ This configuration file has the following settings:
 
     {{< /note >}}
 
-`automatic_attribute_blacklist`
```
Comment (tas50) [inline content/config_rb_client.md:47]: These need to get reverted as well

**Review by tas50 — APPROVED** [review]
ship it

**Comment by kagarmoe** [comment]
I reformatted the file, so the changes will be hard to see. The rewritten parts:

```
`automatic_attribute_blacklist`
: A hash that blocks `automatic` attributes, preventing blocked attributes from being saved. **Deprecated. EOL Chef Infra Client 18 (April 2022)**: Use `blocked_automatic_attributes`.

`automatic_attribute_whitelist`
: A hash that allows `automatic` attributes, preventing non-allowed attributes from being saved. **Deprecated. EOL Chef Infra Client 18 (April 2022)**: Use `allowed_automatic_attributes`.

`normal_attribute_blacklist`
: A hash that blocks `normal` attributes, preventing blocked attributes from being saved. **Deprecated. EOL Chef Infra Client 18 (April 2022)**: Use `blocked_normal_attributes`.

`override_attribute_blacklist`
: A hash that blocks `override` attributes, preventing blocked attributes from being saved. **Deprecated. EOL Chef Infra Client 18 (April 2022)**: Use `blocked_override_attributes`.

`normal_attribute_whitelist`
: A hash that allows `normal` attributes, preventing non-allowed attributes from being saved. **Deprecated. EOL Chef Infra Client 18 (April 2022)**: Use `allowed_normal_attributes`.

`override_attribute_whitelist`
: A hash that allows `override` attributes, preventing non-allowed attributes from being saved. **Deprecated. EOL Chef Infra Client 18 (April 2022)**: Use `override_normal_attributes`.
```

**Comment by IanMadd** [comment]
If settings like `automatic_attribute_blacklist` are already deprecated, does it make sense to list the new setting separately? Like:

`automatic_attribute_blacklist`
    A hash that blocks automatic attributes, preventing blocked attributes from being saved. Deprecated. EOL Chef Infra Client 18 (April 2022): Use blocked_automatic_attributes.
    
`blocked_automatic_attributes`
    A hash that blocks automatic attributes, preventing blocked attributes from being saved. New in Chef Infra Client 17. (version number?)

**Review by IanMadd — APPROVED** [review]

File: content/config_rb_client.md [hunk 3]
```diff
@@ -14,334 +14,192 @@ aliases = ["/config_rb_client.html"]
...
 
     ```ruby
     knife[:authentication_protocol_version] = '1.3'
     ```
 
     {{< note spaces=4 >}}
 
-    Authentication protocol 1.3 is only supported on Chef Server
-    versions 12.4.0 and above.
+    Authentication protocol 1.3 is only supported on Chef Serve versions 12.4.0 and above.
```
Comment (kagarmoe) [inline content/config_rb_client.md]:
```suggestion
    Authentication protocol 1.3 is only supported on Chef Server versions 12.4.0 and above.
```

**Pull request merged**